requirements/cext.txt

```diff
@@ -1 +1 @@
-cryptography==2.0.3
\ No newline at end of file
+cryptography
```

Comment:
It's best to pin the exact versions we need, so that it's easier to build this commit when we go back from the future.

Reply:
The problem is that we use a repository called `piwheels` to get `cryptography` for raspberry pi. `piwheels` only contains `cryptography` with 2.0.3 for now. However, `pypi` now has 2.1 for `cryptography` and some of the 2.0.3 wheels are deleted. So I couldn't get a consistent version of cryptography that works for both `piwheels` and `pypi`. What I do in my code now is to get the latest version of cryptography in both places.

Reply:
piwheels is keeping up with new package releases quite quickly (we were probably just paused (or working through the backlog) when you found we were missing the latest version of cryptography). We have it now: https://www.piwheels.hostedpi.com/simple/cryptography/